tests/integration.spec.ts

```diff
@@ -18,6 +18,8 @@ import path from 'path';
 import fs from 'fs';
 import childProcess from 'child_process';
 
+const validGitignore = 'node_modules/\n/test-results/\n/playwright-report/\n/blob-report/\n/playwright/.cache/\n';
```

Comment:
```suggestion
const validGitignore = `node_modules/
/test-results/
/playwright-report/
/blob-report/
/playwright/.cache/`;
```

Writing it this way might improve readability.